src/intervaloptimisation.jl

```diff
@@ -0,0 +1,98 @@
+#===================================
+Methods using Interval Optimisation
+===================================#
+using .IntervalOptimisation
+
+"""
+    range_and_optimisers(f::Function, dom::Interval_or_IntervalBox;
+                         structure=HeapedVector, tol=1e-3)
```

Comment:
In the docs build:

```julia
ERROR: UndefVarError: HeapedVector not defined
```

Could be related to the some dependency now being optional and hence not loaded in the docs build.

Reply:
yes I think this is the case, I think adding IntervalOptimisation.jl to the docs Project.toml would fix it?